editor/plugins/ot_features_plugin.cpp

```diff
@@ -116,7 +118,19 @@ void OpenTypeFeaturesAdd::setup(Object *p_object) {
 	bool have_ss = false;
 	bool have_cv = false;
 	bool have_cu = false;
-	Dictionary features = Object::cast_to<Control>(edited_object)->get_theme_font(SNAME("font"))->get_feature_list();
+
+	Dictionary features;
+
+	Control *ctrl = Object::cast_to<Control>(edited_object);
+	if (ctrl != nullptr) {
+		features = ctrl->get_theme_font(SNAME("font"))->get_feature_list();
+	} else {
+		Label3D *l3d = Object::cast_to<Label3D>(edited_object);
+		if (l3d != nullptr) {
+			features = l3d->get_font()->get_feature_list();
```

Comment:
This line crashes when I create a new `Label3D`, the new label has a null `Font`.